packages/atomic/src/components/commerce/product-template-components/product-utils.ts

```diff
@@ -12,3 +13,37 @@ export function getStringValueFromProductOrNull(
 
   return value;
 }
+
+export function buildStringTemplateFromProduct(
+  template: string,
+  product: Product,
+  bindings: CommerceBindings
+) {
+  return template.replace(/\${(.*?)}/g, (value: string) => {
+    const key = value.substring(2, value.length - 1);
+    let newValue = readFromObject(product, key);
+    if (!newValue) {
+      newValue = readFromObject(window, key);
```

Comment:
Not sure I understand `window` here. Can you explain ?

Reply:
The docs for hrefTemplate indicates what it is used for
>  The template literal can reference any number of product properties from the parent product. **It can also reference the window object.**